### ISSUE TITLE
Dashboard won't load (timeout) on older device (iPad mini 2)

### ISSUE THREAD

**Issue by DChiuch** [AalianKhan/mushroom-strategy#97]
**Describe the bug**  
Since version mushroom strategy 1.0.0, I'm unable to use the mushroom strategy dashboard on my iPad mini 2. It used to work on mushroom strategy version 0.15. This happens in both the Home Assistant iOS app, and in the browser.

**Versions**  
* Mushroom-Strategy: 1.0.0+
* HACS: 1.33.0
* Mushroom: 3.2.2
* Home Assistant: 2023.11.2

**Screenshots**  
<img width="1075" alt="image" src="https://github.com/AalianKhan/mushroom-strategy/assets/6777430/f824bafc-b2d0-43ff-9c67-903ee9d3ee34">

Error loading the dashboard strategy:
Error: Timeout waiting for strategy element ll-strategy-dashboard-mushroom-strategy to be registered

**Comment by DigiLive** [comment]
Please report the output of your browsers' debug console.
(https://help.remo.co/en/support/solutions/articles/63000251570-how-to-activate-the-iphone-debug-console-or-web-inspector-)

Also, whats your IOS version?
Currently we only support IOS 15.4 and above.
The next release should be less cumbersome regarding the IOS version.

**Comment by DChiuch** [comment]
```
[Error] SyntaxError: Invalid character: '#'
[Info] 🍄 Mushroom 🍄 - 3.2.2 (mushroom.js, line 3557)
[Info]  MINI-GRAPH-CARD  0.11.0  (mini-graph-card-bundle.js, line 1)
[Info]    XIAOMI-VACUUM-MAP-CARD    (xiaomi-vacuum-map-card.js, line 1080)
   Version v2.2.2           
[Error] Unhandled Promise Rejection: https://xxx.duckdns.org/hacsfiles/mushroom-strategy/mushroom-strategy.js?hacstag=619284862101
	(anonymous function)
	rejectPromise
[Error] Error: Timeout waiting for strategy element ll-strategy-dashboard-mushroom-strategy to be registered — wrap-error-constructor-with-cause.js:36
	(anonymous function) (20251-p0IazryQO-o.js:1:49432)
	f (app-WhhiEP-dtFE.js:2:141463)
	(anonymous function) (app-WhhiEP-dtFE.js:2:142812)
	r (app-WhhiEP-dtFE.js:2:128201)
	c (app-WhhiEP-dtFE.js:2:128441)
	promiseReactionJob
```

iOS version is 12.5.7

**Comment by DigiLive** [comment]
Thank you...
Theoretically I've identified your issue.
It is caused by missing features in iOS/iPadOS < 14.4.

Small changes upcoming releases gonna support 6 to 10yo devices with a global usage of < 1%.
Probably the latest commit we have available which will still work is de70fddddd3da8bf7b3d3b34930285f54e54da28, but will be missing many fixes and added features.
Your best bet is to see if there are any other browsers available in the App store and try them out.
